packages/fluentui/react-northstar/src/components/Chat/chatLayoutContext.ts

```diff
@@ -0,0 +1,9 @@
+import { createContext, useContext } from 'react';
+
+export type chatLayout = 'comfy' | 'compact';
```

Comment:
nit: types should be pascal case
```suggestion
export type ChatLayout = 'comfy' | 'compact';
```